lib/atomic_web/components/dropdown.ex

```diff
@@ -0,0 +1,59 @@
+defmodule AtomicWeb.Components.Dropdown do
+  @moduledoc false
+  use AtomicWeb, :component
+
+  import AtomicWeb.Components.Icon
+
+  attr :label, :string, required: true, doc: "The label of the dropdown."
+  attr :image, :string, default: "", doc: "The image of the dropdown."
+
+  attr :type, :atom,
+    default: :button,
+    values: [:button, :avatar],
+    doc: "The type of the dropdown."
+
+  attr :items, :list, default: [], doc: "The items to display in the dropdown."
+
+  def dropdown(%{type: :avatar} = assigns) do
+    ~H"""
+    <div class="relative inline-block text-left">
+      <button id={"dropdown-#{assigns.label}"} type="button" phx-hook="Dropdown" class="inline-flex h-10 w-10 justify-center rounded-3xl border border-gray-300 bg-white px-4 py-2 text-sm font-medium text-gray-700 shadow-sm hover:bg-gray-50">
```

Comment:
No need for hooks, use [LiveView JS](https://hexdocs.pm/phoenix_live_view/0.19.5/Phoenix.LiveView.JS.html#toggle_attribute/1)!
```suggestion
      <button id="dropdown-button" type="button" phx-click={JS.toggle(to: "#dropdown-#{assigns.label}", in: "block", out: "hidden")} class="inline-flex justify-center rounded border border-gray-300 bg-white px-4 py-2 text-sm font-medium text-gray-700 shadow-sm hover:bg-gray-50">
```